.github/workflows/validate-datafiles.yml

```diff
@@ -16,4 +16,5 @@ jobs:
       - uses: actions/checkout@v3
       - uses: DeterminateSystems/nix-installer-action@main
       - uses: DeterminateSystems/magic-nix-cache-action@main
-      - run: nix build -L .#checks.pytest-facts
+      - run: nix build -L .#checks.x86_64-linux.pytest-facts
```

Comment:
if you run `nix flake check` it will evaluate all the code in the repo, even nixos configurations and build all derivations under the `checks` output.

Reply:
agreed, but I think itll have an issue running all tests due to the need for qemu in github actions